applications: nrf_desktop: Fix finding next report state (#6024)
nrfconnect/sdk-nrf: doc/nrf/known_issues.rst

.. rst-class:: v1-7-1 v1-7-0

> NCSDK-12337: Possible assertion fail while a USB connected host boots

Reviewer: ```suggestion
NCSDK-12337: Possible assertion at boot of an USB-connected host
```

>   While the USB connected host boots, the HID report subscriptions may be disabled and enabled a few times without disconnecting the USB.

Reviewer: ```suggestion
 During the booting procedure of a host device connected through USB, the HID report subscriptions might be disabled and enabled a few times without disconnecting the USB.
```

>   That may result in improper subscription handling and assertion failure in :ref:`nrf_desktop_hid_state`.

Reviewer: ```suggestion
  This can result in improper subscription handling and assertion failure in the :ref:`nrf_desktop_hid_state`.
```

>   **Workaround:** Manually cherry-pick and apply commit with fix from main.

Reviewer: Which commit should be cherry-picked? Can you provide the SHA here?

Author: I will merge commit with fix first and then we will add a known issue (currently I cannot refer to commit hash, because fix is not merged yet).

Author: Added
